lib/inets/test/httpc_SUITE.erl

```diff
@@ -2076,7 +2076,7 @@ server_config(http_internal, Config) ->
     ];
 server_config(https, Config) ->
     Ssl = proplists:get_value(test_ssl, Config, ssl),
-    [{socket_type, {Ssl, ssl_config(Config)}} | server_config(http, Config)];
+    [{socket_type, {Ssl, ssl_config(Config)}} | server_config(http, [{test_ssl, essl} | Config] )];
```

Comment:
if `Ssl =:= ssl` we get an error. This happens in `maint` as well.
If `Ssl =:= essl`, then all tests pass

Reply:
@IngelaAndin Do you know if this behaviour is expected in `maint`?

Reply:
You probably ran the modified test case with maint!? I do not see it failing in the nightly builds on maint.